src/xdocs/config_imports.xml

```diff
@@ -881,32 +881,56 @@ public class InputIllegalImport { }
       <subsection name="Description">
         <p>Since Checkstyle 4.0</p>
         <p>
-          Controls what can be imported in each package. Useful for
+          Controls what can be imported in each package and compilation unit. Useful for
           ensuring that application layering rules are not violated,
           especially on large projects.
         </p>
+        <p>
+          You can control imports based on the a package name or based on the compilation unit
+          name. When controlling packages, all compilation units and sub-packages in the declared
+          package will be controlled by this check. To specify differences between a main package
+          and a sub-package, you must define the sub-package inside the main package. When
+          controlling compilation units, only the first, top-level compilation unit declared in
```

Comment:
let use here "file name". We need to be clear to user that in config they define "class" to allow/disallow (that actually a type). But when they define unit, unit is file, so unit name is file name.

Reply:
Same discussion as https://github.com/checkstyle/checkstyle/pull/5641#discussion_r181582833 .